themes/base/css/storyTelling.css

```diff
@@ -381,11 +381,16 @@ img.sjrk-st-block-media-preview:not([src]) {
     background: url(../img/icons/Image-background.svg) center center no-repeat;
 }
 
-video.sjrk-st-block-media-preview[src=""],
-video.sjrk-st-block-media-preview:not([src]) {
+video.sjrk-st-block-media-preview[src=""] + .sjrk-st-block-video-preview-placeholder,
+video.sjrk-st-block-media-preview:not([src]) + .sjrk-st-block-video-preview-placeholder {
     background: url(../img/icons/Video-background.svg) center center no-repeat;
 }
 
+video:not([controls]) {
+  display: none;
+  height: 0;
```

Comment:
Is `height` required here? `display: none;` should remove the element visually, layout spacing, and from the a11y tree.

Reply:
This is a good point. I'd left it in there as this is what Normalize does with `audio` elements, but since this rule isn't even present in Normalize after 7.x, I've removed the `height` specification and added an `audio:not([controls])` selector to this rule, to ensure it continues to work as expected if and when we update Normalize